CHANGELOG.md

```diff
@@ -10,17 +10,16 @@ The versions in this change log should match those published
 to [the Sonatype Maven Central Repository][].
 It is those war files that are being versioned.
 
-## Next (18.1.0)
+## Next (18.1.1)
 
-+ Re-adding myuw-feedback web component with conditional flag feature
-
-## 18.0.1 - 2021-01-14
```

Comment:
This diff on CHANGELOG looks goofy. [18.0.1 was released](https://repo1.maven.org/maven2/org/apereo/uportal/uportal-app-framework/18.0.1/) so ideally we'd keep around an entry for it in the changelog.

Reply:
I updated the changelog.